.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: php
 
 php:
```

Comment:
Is this still necessary with the addition of the `matrix` below?

Reply:
Yep, they work together rather than exclusively

Reply:
Neat :)